Libraries/LibC/net/route.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020, Andreas Kling <kling@serenityos.org>
```

Comment:
I didn't write this file :)

Reply:
I thought since the file is 95% the license your name would be more fitting.